.github/workflows/greetings.yml

```
@@ -9,5 +9,5 @@ jobs:
     - uses: actions/first-interaction@v1
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        issue-message: 'Hi and welcome! Thanks for posting your first issue in the PyVista project! Someone from @pyvista/developers will chime in before too long. If your question is support related, it may be automatically transferred to https://github.com/pyvista/pyvista-support'
+        issue-message: 'Hi and welcome! Thanks for posting your first issue in the PyVista project! Someone from @pyvista/developers will chime in before too long. If your question is support related, it may be automatically transferred to https://github.com/pyvista/pyvista/discussions'
```

Mild suggestion to make the link a bit prettier. I know it was a raw link to begin with, so it can be either way.
```suggestion
        issue-message: 'Hi and welcome! Thanks for posting your first issue in the PyVista project! Someone from @pyvista/developers will chime in before too long. If your question is support related, it may be automatically transferred to [Discussions](https://github.com/pyvista/pyvista/discussions).'
```

---

Good point.  Thanks!